qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -123,62 +124,169 @@ def calc_xdata(length_vector, length_multiplier):
             the previous sequences.
-        rb_pattern: A list of the form [[i,j],[k],...] which will make
-            simultaneous RB sequences where
-            Qi,Qj are a 2Q RB sequence and Qk is a 1Q sequence, etc.
-            E.g. [[0,3],[2],[1]] would create RB sequences that are
-            2Q for Q0/Q3, 1Q for Q1+Q2
-            The number of qubits is the sum of the entries.
-            For 'regular' RB the qubit_pattern is just [[0]],[[0,1]].
-        length_multiplier: if this is an array it scales each rb_sequence by
-            the multiplier
-        seed_offset: What to start the seeds at (e.g. if we
-            want to add more seeds later)
-        align_cliffs: If true adds a barrier across all qubits in rb_pattern
-            after each set of elements, not necessarily Cliffords
-            (note: aligns after each increment of elements including the
-            length multiplier so if the multiplier is [1,3] it will barrier
-            after 1 element for the first pattern and 3 for the second).
-        interleaved_gates: A list of gates of elements that
-            will be interleaved (for interleaved randomized benchmarking)
-            The length of the list would equal the length of the rb_pattern.
-        is_purity: True only for purity rb (default is False)
-        group_gates: On which group (or gate set) we perform RB
-            (default is the Clifford group)
-            '0' or None or 'Clifford': Clifford group
-            '1' or 'CNOT-Dihedral' or 'Non-Clifford': CNOT-Dihedral group
 
+            For example:
+
+            .. code-block::
+
+                length_vector = [1, 10, 20, 50, 75, 100, 125, 150, 175]
+
+            ``length_vector = None`` is the same as ``length_vector = [1, 10, 20]``
+
+        rb_pattern: A list of the lists of integers representing the
+            qubits indexes. For example, ``[[i,j],[k],...]`` will make
+            simultaneous RB sequences, where
+            there is a 2-qubit RB sequence on qbits Qi and Qj,
+            and a 1-qubit RB sequence on qubit Qk, etc.
+            Each qubit appers at most once.
+            The number of qubits on which RB is done is the sum of the lists
+            sizes.
+
+            For example:
+
+            * ``rb_pattern = [[0]]`` or ``rb_pattern = None``
+
+            create a 1-qubit RB sequence on qubit Q0.
+
+            * ``rb_pattern = [[0,1]]``
+
+            create a 2-qubit RB sequence on qubits Q0 and Q1.
+
+            * ``rb_pattern = [[0,3],[2],[1]]``
+
+            create RB sequences that are 2-qubit RB for qubits Q0 and Q3,
+            1-qubit RB for qubit Q1, and 1-qubit RB for qubit Q2.
+
+            * ``rb_pattern = [[2],[6,4]]``
+
+            create RB sequences that are 2-qubit RB for qubits Q6 and Q4,
+            and 1-qubit RB for qubit Q2.
+
+        length_multiplier: An array that scales each RB sequence by
+            the multiplier.
+
+            For example:
```

Comment:
Having these code examples in the args documentation makes it very difficult to read.

I think all these kind of detailed descriptions and examples should be included in the general function documentation and the `Args:` and `Returns:` documentation should just be a shorter deescription.

```
Args:
    arg: short description or arg.

Returns:
    short description of return.
```

Reply:
OK, I moved most of the example to the Example section at the end of the doc.
I only left a few small examples inside the args.
I followed the documentation of:
https://qiskit.org/documentation/_modules/qiskit/circuit/quantumcircuit.html#QuantumCircuit